python-avd/pyavd/_eos_cli_config_gen/schema/eos_cli_config_gen.schema.yml

```diff
@@ -10256,6 +10256,19 @@ keys:
               type: str
               convert_types:
               - int
+            tls:
```

Comment:
Add in the description that when `tls` is configured then `key` is ignored as they are mutually exclusive in the CLI line? (Add it in the `key` description as well)

Reply:
Added description to both `tls` and `key`.